### PR TITLE
description change and doc changes

### LINKED ISSUE #43: Document adding SUPPORTED_DISTROS to config.py
After importing the JSON files in step 5 of the documentation, you also need to update config.py to add them under SUPPORTED_DISTROS (ZOS is the only one there by default). This should be documented.

Also, I just noticed there are now two steps labeled Step 6 - oops! Should fix this up while you're in there adding a step anyway :smile:

### LINKED ISSUE #44: z/OS name formatting
Currently in the UI, z/OS is called ZOS. We want to be using the proper capitalization and include the forward slash, since that reflects what users will expect the operating system to be called.

### LINKED ISSUE #45: Support "Description" for sources without descriptions
In #42 we got support for a description column :tada: but now when we try to import JSON sources that do not include the Description column (including all of the legacy PDS sources, which will never include a Description column), an error is thrown:

```
[Tue Jul 06 17:55:37.944969 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx] mod_wsgi (pid=1720): Failed to exec Python script file '/var/www/sdt.princessleia.com/htdocs/src/SDT.wsgi'.
[Tue Jul 06 17:55:37.945150 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx] mod_wsgi (pid=1720): Exception occurred processing WSGI script '/var/www/sdt.princessleia.com/htdocs/src/SDT.wsgi'.
[Tue Jul 06 17:55:37.946432 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx] Traceback (most recent call last):
[Tue Jul 06 17:55:37.946510 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]   File "/var/www/sdt.princessleia.com/htdocs/src/classes/package_search.py", line 54, in loadPackageData
[Tue Jul 06 17:55:37.946518 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]     json_data = json.load(open(distro_data_file))
[Tue Jul 06 17:55:37.946528 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]   File "/usr/lib/python3.7/json/__init__.py", line 296, in load
[Tue Jul 06 17:55:37.946533 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]     parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
[Tue Jul 06 17:55:37.946542 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]   File "/usr/lib/python3.7/json/__init__.py", line 348, in loads
[Tue Jul 06 17:55:37.946547 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]     return _default_decoder.decode(s)
[Tue Jul 06 17:55:37.946554 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]   File "/usr/lib/python3.7/json/decoder.py", line 337, in decode
[Tue Jul 06 17:55:37.946559 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
[Tue Jul 06 17:55:37.946570 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]   File "/usr/lib/python3.7/json/decoder.py", line 355, in raw_decode
[Tue Jul 06 17:55:37.946578 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]     raise JSONDecodeError("Expecting value", s, err.value) from None
[Tue Jul 06 17:55:37.946609 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx] json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
[Tue Jul 06 17:55:37.946647 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]
[Tue Jul 06 17:55:37.946653 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx] During handling of the above exception, another exception occurred:
[Tue Jul 06 17:55:37.946658 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]
[Tue Jul 06 17:55:37.946666 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx] Traceback (most recent call last):
[Tue Jul 06 17:55:37.946709 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]   File "/var/www/sdt.princessleia.com/htdocs/src/SDT.wsgi", line 5, in <module>
[Tue Jul 06 17:55:37.946715 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]     from main import app as application
[Tue Jul 06 17:55:37.946721 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]   File "/var/www/sdt.princessleia.com/htdocs/src/main.py", line 11, in <module>
[Tue Jul 06 17:55:37.946724 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]     package_search = PackageSearch.load()
[Tue Jul 06 17:55:37.946732 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]   File "/var/www/sdt.princessleia.com/htdocs/src/classes/package_search.py", line 132, in load
[Tue Jul 06 17:55:37.946736 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]     return cls.get_instance()
[Tue Jul 06 17:55:37.946743 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]   File "/var/www/sdt.princessleia.com/htdocs/src/classes/package_search.py", line 123, in get_instance
[Tue Jul 06 17:55:37.946747 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]     cls.INSTANCE.package_data = cls.loadPackageData()
[Tue Jul 06 17:55:37.946754 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]   File "/var/www/sdt.princessleia.com/htdocs/src/classes/package_search.py", line 58, in loadPackageData
[Tue Jul 06 17:55:37.946759 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]     json_data = cls.preparePackageData()
[Tue Jul 06 17:55:37.946766 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]   File "/var/www/sdt.princessleia.com/htdocs/src/classes/package_search.py", line 93, in preparePackageData
[Tue Jul 06 17:55:37.946769 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx]     cachedPackage["D"] = pkg["description"]
[Tue Jul 06 17:55:37.946784 2021] [wsgi:error] [pid 1720] [remote xxxx:xxxx] KeyError: 'description'
```

### PR DESCRIPTION
**Pull request by rachejazz**
Signed-off-by: rachejazz <divyadeepti2000@gmail.com>

- fixes #43 
- fixes #44  

image

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -140,6 +140,8 @@ Thanks for using SDT!
  _**NOTE:**_
 - Make sure the json file exists in the PDS data directory.
 
+Now to know how to update the `src/config/config.py` to reflect the new json files in the UI, follow steps mentioned in [Adding_new_distros](Adding_new_distros.md)
```
Comment (pleia2) [inline docs/Installation.md:143]: So people know where to go, can you either mention Step 2 specifically, and/or link directly to that section of the doc? https://github.com/openmainframeproject/software-discovery-tool/blob/master/docs/Adding_new_distros.md#step-2-update-the-supported_distros-variable-in-configuration-file-pds_basesrcconfigconfigpy

File: src/config/config.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
 DEBUG_LEVEL = logging.ERROR
 
 SUPPORTED_DISTROS = {
-	'ZOS': {
+	'IBM z/OS ™': {
 		'ZOS Software': 'ZOS_Software_List.json'
```
Comment (pleia2) [inline src/config/config.py:38]: z/OS here too.

**Review by pleia2 — CHANGES_REQUESTED** [review]

File: src/config/config.py [hunk 3]
```diff
@@ -36,6 +36,9 @@
 SUPPORTED_DISTROS = {
 	'IBM z/OS ™': {
 		'ZOS Software': 'ZOS_Software_List.json'
+	},
+	'Debian': {
+		'Buster': 'Debian_Buster_List.json'
```
Comment (pleia2) [inline src/config/config.py:41]: Strictly speaking, Buster is the code name for development, so we probably want to refer to it by the official version number, this is how the Debian project refers to it: https://www.debian.org/News/2019/20190706

Comment (pleia2) [reply src/config/config.py:41]: Debian 10 "buster"

Comment (rachejazz) [reply src/config/config.py:41]: Ohh cool I'll do that

File: docs/Installation.md (hunk 1 above)
Comment (rachejazz) [reply docs/Installation.md:143]: Sure okkk

**Review by pleia2 — APPROVED** [review]

**Pull request merged**